components/admin_console/billing/billing_subscriptions/index.tsx

```diff
@@ -63,6 +67,46 @@ const BillingSubscriptions: React.FC<Props> = () => {
 
     const [showCreditCardBanner, setShowCreditCardBanner] = useState(true);
 
+    const product = useSelector((state: GlobalState) => {
+        const products = state.entities.cloud.products!;
+        if (!products) {
+            return null;
+        }
+        const keys = Object.keys(products);
+        let product: Product;
+        if (products && subscription) {
+            product = products[subscription?.product_id];
+            if (!product) {
+                keys.forEach((key) => {
+                    if (products[key].name.toLowerCase().includes('professional')) {
+                        product = products[key];
+                    }
+                });
+            }
+            if (product) {
+                return product;
+            }
+        }
+        return products[keys[0]];
+    });
+
+    let subscriptionPlan = 'CLOUD_PROFESSIONAL';
+
+    switch (product?.sku) {
+    case 'cloud-starter':
+        subscriptionPlan = 'CLOUD_STARTER';
+        break;
+    case 'cloud-professional':
+        subscriptionPlan = 'CLOUD_PROFESSIONAL';
+        break;
+    case 'cloud-enterprise':
+        subscriptionPlan = 'CLOUD_ENTERPRISE';
+        break;
+    default:
+        subscriptionPlan = 'CLOUD_PROFESSIONAL';
+        break;
+    }
```

Comment:
Now that we have the SKU's in a more code-friendly manner, is this switch statement needed? We could just pass the SKU as the subscriptionPlan, and then use the SKU names `cloud-starter` `cloud-professional` etc here: https://github.com/mattermost/mattermost-webapp/pull/8094/files#diff-24ded2951469e8a6dd5fa613f85043cb6c88fcb5fdd9815b90cb8ad704cf9905R40

Reply:
+1 on this approach 👍

Reply:
yes, it is changed already!